Readme.md

```diff
@@ -32,15 +32,17 @@ Accepts an AST `object` (as `css.parse` produces) and returns a CSS string.
 
 `options`:
 
-- compress: omit comments and extraneous whitespace.
-- sourcemap: return a sourcemap along with the CSS output. Using the `source`
+ - `compress`: omit comments and extraneous whitespace.
```

Comment:
I think all option names used to be wrapped in backticks but that it was removed in a specific commit.